src/components/Task/Modal/Form/Inputs/SuggestedContactStatus/SuggestedContactStatus.test.tsx

```diff
@@ -114,16 +114,16 @@ describe('SuggestedContactStatus', () => {
     );
     await waitFor(() => {
       expect(mutationSpy).toHaveBeenCalledTimes(2);
-      expect(mutationSpy.mock.calls[0][0].operation).toMatchObject({
-        operationName: 'ContactStatus',
-        variables: {
-          accountListId: accountListId,
-          contactId: 'contact-1',
-        },
+      expect(mutationSpy).toHaveGraphqlOperation('ContactStatus', {
+        accountListId: accountListId,
+        contactId: 'contact-1',
       });
     });
 
-    expect(await findByText('Initiate for Appointment')).toBeInTheDocument();
+    await waitFor(
```

Comment:
You can also customize `findByText`'s timeout if you need to: `findByText('...', {}, { timeout: 3000 })`.

Reply:
Ah ok good to know